### PR TITLE
[2.02] Create a human-readable output for sum rules

### PR DESCRIPTION
**Pull request by hayfield**
This is shown on pages such as http://iatistandard.org/201/rulesets/standard-ruleset/

Note: This has not been tested. It is assumed to work first time.

### CODE REVIEW

File: iatirulesets/text.py [hunk 1]
```diff
@@ -33,6 +33,12 @@ def rules_text(rules, reduced_path, show_all=False):
                             out.append('``{0}`` should start with the value in ``{1}``'.format(case_path, case['start']))
                         elif rule == 'regex_matches':
                             out.append('``{0}`` should match the regex ``{1}``'.format(case_path, case['regex']))
+                        elif rule == 'sum':
+                            sum_total = case['sum']
+                            if other_paths:
+                                out.append('The sum of values matched by ``{0}`` and ``{1}`` must be ``{2}``.'.format(case_path, human_list(other_paths, 'and'), sum_total))
+                            else:
+                                out.append('The sum of values matched by ``{0}`` must be ``{2}``.'.format(case_path, sum_total))
```
Comment (dalepotter) [inline iatirulesets/text.py]: In terms of copy, how about this text for added clarity - either
`The sum of values matched at paths [path] and [path] must be [sum_total]`
`The sum of values contained within [path] and [path] must be [sum_total]`

??

Comment (dalepotter) [inline iatirulesets/text.py]: There might be other suggestions too for improving the clarity of this sentence, when it appears on the website.

Comment (hayfield) [inline iatirulesets/text.py]: Feel that your first suggestion is my favourite.

@IATI/bas - any thoughts?

**Comment by hayfield** [comment]
@dalepotter It seems that the changes need approving here as well as in #27

**Comment by hayfield** [comment]
Will merge this after a test generation of the 2.01 PR (along with the IATI/IATI-Standard-SSOT#92 changes happening on 1.04).

**Pull request merged**